ava-labs/core-extension
# chore: update SDKs

## Description
* Updates `@avalabs/core-wallets-sdk` (and other SDKs), therefore I had to fix some backwards incompatibility issues
* Updates `@avalabs/evm-module` (and other VM modules) to fix [CP-9374](https://ava-labs.atlassian.net/browse/CP-9374)

## Testing
* Deploy an L1 with AvaCloud
* Deploy Multicall3 contract on it (it should succeed)

## Screenshots:

https://github.com/user-attachments/assets/ffce3804-edad-4cc5-8279-196d4c1df038

☝️ _the error toast you see at the beginning is just me being too fast, it seems the API was not ready to respond yet (I recieved `404 Not Found` and `503 Service Unavailable` responses)_

## Checklist for the author
